spec/features/show_work_in_collection_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe "In Show Collection", :type => :feature do
+  let(:user) { create :jill }
+
+  let!(:work) {
```

Comment:
@tingtingjh you can create a public work using the Factory: let(work) { FactoryGirl.create(:public_generic_work) }